visionipc/visionipc_tests.cc

```diff
@@ -22,6 +22,17 @@ TEST_CASE("Connecting"){
   REQUIRE(client.connected);
 }
 
+TEST_CASE("getAvailableStreams"){
+  VisionIpcServer server("camerad");
+  server.create_buffers(VISION_STREAM_ROAD, 1, false, 100, 100);
+  server.create_buffers(VISION_STREAM_WIDE_ROAD, 1, false, 100, 100);
+  server.start_listener();
+  auto available_streams = VisionIpcClient::getAvailableStreams("camerad");
+  REQUIRE(available_streams.size() == 2);
+  REQUIRE(available_streams[0] == VISION_STREAM_ROAD);
+  REQUIRE(available_streams[1] == VISION_STREAM_WIDE_ROAD);
```

Comment:
does it make sense to return an std::set instead to better write things like this?